src/components/button/button.stories.jsx

```diff
@@ -0,0 +1,76 @@
+import emailIcon from "../../images/Email.svg";
+import playIcon from "../../images/Play.svg";
+import Button from "./button";
+
+export default {
+  title: "Button",
+  component: Button,
+};
+//email support button
+export const EmailButton = ({ onClick }) => {
+  return (
+    <Button onClick={onClick} style="border rounded-lg overflow-hidden">
```

Comment:
Could you refactor this slightly--

I'd like to have each of these buttons inside of `button.jsx` instead of here, so that you can re-use them outside of storybook. My intention is that it would look something like this:

```
Button.withIcon = ({ icon, children, ...rest }) => {
  return (
    <Button style="border rounded-lg overflow-hidden" {...rest}>
      <div className="bg-black/5 px-9 py-1 flex flex-col items-center duration-300 hover:bg-black/10">
        <img src={icon} className="w-6 h-6" />;
        {children}
      </div>
    </Button>
  );
};
```

And then the storybook file would use that component. Basically we just need to move this code to the other file and refactor slightly. For the `<p>` that's inside of this button, you should be able to remove that tag and apply those text styles to the button itself